src/utils/index.tsx

```diff
@@ -41,7 +41,7 @@ export function createContext<ContextType>(options: CreateContextOptions = {}) {
       throw new Error(errorMessage);
     }
 
-    return context;
+    return context || {};
```

Comment:
Seems like a bug with our createContext

When strict is false and we try to use any component it throws error because the useContext hook cannot destructure undefined value.